packages/containerd-2.1/etc-containerd.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Containerd Configuration Directory (/etc/containerd)
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
+After=selinux-policy-files.service
+Wants=selinux-policy-files.service
+
+[Mount]
+What=tmpfs
+Where=/etc/containerd
+Type=tmpfs
+Options=nosuid,nodev,noexec,noatime,mode=0750,context=system_u:object_r:secret_t:s0
```

Comment:
We should make this change in newer versions of containerd and k8s:
```suggestion
Options=nosuid,nodev,noexec,noatime,mode=0750,context=system_u:object_r:etc_secret_t:s0
```

(The k8s equivalent is in `etc-kubernetes-pki-private.mount`).